<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServiceUtilities</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ServiceUtilities.qrc">
    <normaloff>:/icon/app.png</normaloff>:/icon/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="toolTip">
         <string>New service</string>
        </property>
        <property name="icon">
         <iconset resource="ServiceUtilities.qrc">
          <normaloff>:/icon/plus.png</normaloff>:/icon/plus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="toolTip">
         <string>Remove service</string>
        </property>
        <property name="icon">
         <iconset resource="ServiceUtilities.qrc">
          <normaloff>:/icon/minus.png</normaloff>:/icon/minus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="toolTip">
         <string>Edit service</string>
        </property>
        <property name="icon">
         <iconset resource="ServiceUtilities.qrc">
          <normaloff>:/icon/gear.png</normaloff>:/icon/gear.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="reloadButton">
        <property name="toolTip">
         <string>Reload services</string>
        </property>
        <property name="icon">
         <iconset resource="ServiceUtilities.qrc">
          <normaloff>:/icon/reload.png</normaloff>:/icon/reload.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputButton">
        <property name="toolTip">
         <string>Show output window</string>
        </property>
        <property name="icon">
         <iconset resource="ServiceUtilities.qrc">
          <normaloff>:/icon/console.png</normaloff>:/icon/console.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>100</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ServiceUtilities.qrc"/>
 </resources>
 <connections/>
</ui>
